pages/linux/nmcli-connection.md

```diff
@@ -26,3 +26,7 @@
 - Create a static IPv4-only connection:
 
 `nmcli connection add ifname {{interface_name}} type {{ethernet}} ip4 {{10.0.0.7/8}} gw4 {{10.0.0.1}} ipv4.dns {{10.0.0.1}} ipv6.method {{ignore}}`
+
+- Create a VPN connection using OpenVPN from a OVPN file:
+
+`nmcli connection import type openvpn file {{vpn_file}}.ovpn`
```

Comment:
No path?

Reply:
Definitely should be one!

Reply:
If the extension in not mandatory should be inside the token syntax, also the type is an argument option:
```suggestion
`nmcli connection import type {{openvpn}} file {{path/to/vpn_config.ovpn}}`
```

Reply:
This example is specifically for `openvpn`, so I don't think it should go in token syntax

Reply:
There are plugins for `openconnect` and others, in `gh-issue` the example is `list 10 GitHub issues` the example is `--limit {{10}}` because is just an argument options like the VPN type.